recipes/firefly-client/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "firefly-client" %}
+{% set version = "2.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 4e200b8e3c8399206b9e10af386fe5f8aefa5dc77f3bc48b9182f570ebba70ec
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+
+  run:
+    - python
+    - future
+    - requests
+    - setuptools
+    - ws4py
+
+test:
+  imports:
+    - firefly_client
+
+about:
+  home: http://github.com/Caltech-IPAC/firefly_client
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: License.txt
```

Comment:
Add a comment above the `license_file` line:

```
# license file manually packaged. See: https://github.com/Caltech-IPAC/firefly_client/pull/42
license_file: License.txt
...
```

This will be a reminder to check if the license file is included in the source distribution in future releases